### ISSUE TITLE
Selenium Grid Node crashes on startup when attempting to start node without -browser flag

### ISSUE THREAD

**Issue by krmahadevan** [SeleniumHQ/selenium#2540]
## Meta -
OS:  OSX
 
Selenium Version:  '3.0.0-beta1', revision: '8e4315c'

Browser:  Not related to a specific browser

Browser Version:  Not related to a specific browser

## Expected Behavior -

Node should spawn and get hooked to the Grid.

## Actual Behavior -

The node JVM crashes with the below stacktrace.
```

java.lang.NullPointerException
	at org.openqa.grid.common.RegistrationRequest.build(RegistrationRequest.java:186)
	at org.openqa.grid.selenium.GridLauncherV3$3.launch(GridLauncherV3.java:242)
	at org.openqa.grid.selenium.GridLauncherV3.main(GridLauncherV3.java:85)
```

I believe the bug is due to this commit : https://github.com/SeleniumHQ/selenium/commit/55e987831d2550e57451b294262cd16b343d22da

## Steps to reproduce -
Download the latest released version of Selenium server standalone [3.0.0-beta1] and try spawning a Grid/Node setup using the below commands

Grid :
`java -jar selenium-server-standalone-3.0.0-beta1.jar -role hub`

Node : 
`java -jar selenium-server-standalone-3.0.0-beta1.jar -role node -hub http://localhost:4444/grid/register/`

**Comment by lukeis** [comment]
Thanks for the issue report :)

Already fixed with https://github.com/SeleniumHQ/selenium/commit/afebab51ab353e13f80b1fd21c6aca26114a764c

**Comment by krmahadevan** [comment]
My bad. I should have double checked to see if this was fixed already before raising a PR. Guess got a but too excited on sending a PR. 


Thanks & Regards

Krishnan Mahadevan


"All the desirable things in life are either illegal, expensive, fattening or in love with someone else!"




On Sun, Jul 31, 2016 at 5:37 AM +0530, "Luke Inman-Semerau" <notifications@github.com> wrote:












Thanks for the issue report :)



Already fixed with afebab5
